client/src/App.tsx

```diff
@@ -23,9 +22,10 @@ import {
   UpdateGalleryContextProvider,
   StatsContextProvider,
 } from './context';
+import { ThemeProviderWrapper } from './context/ThemeContext';
 import { AuthProvider } from './context/AuthContext';
 
-const socket = io.connect('http://localhost:8080');
+const socket = io.connect('http://3.75.142.65/');
```

Comment:
what is the benefit of proxy if the backend service found her?
include the url in env variables.